frontend/src/pages/FormStaffPage.vue

```diff
@@ -365,7 +282,7 @@ const onBack = () => {
     progressData[currentTab.value + 1].kind = "incomplete";
     progressData[currentTab.value].kind = "current";
     setScrollPoint("step-title");
-    setTimeout(revalidateBus.emit, 1000);
+    setTimeout(() => revalidateBus.emit(progressData[currentTab.value].fields), 1000);
```

Comment:
Again, it seems like progressData[currentTab.value].fields is undefined. Is it intended?